dist/bemmer.js

```diff
@@ -3,89 +3,133 @@
 Object.defineProperty(exports, '__esModule', {
   value: true
 });
-var DEFAULT_ELEMENT_PREFIX = '__';
-var DEFAULT_MODIFIER_PREFIX = '--';
-var SPLITTER_REGEXP = /[\-\_\.\*\+\:\;\/]{2,}/g;
 
-var elementPrefix = DEFAULT_ELEMENT_PREFIX;
-var modifierPrefix = DEFAULT_MODIFIER_PREFIX;
+var _createClass = (function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ('value' in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; })();
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'descriptor'.<br>Expected '{' and instead saw 'defineProperties'.